polus-apply-flatfield-plugin/README.md

```diff
@@ -0,0 +1,47 @@
+# Apply Flatfield
+
+This WIPP plugin applies a flatfield operation on every image in a collection. The algorithm used to apply the flatfield is as follows:
+
+$$Corrected = \frac{Original - Darkfield}{Brightfield} - Photobleach + Offset$$
```

Comment:
Have you been able to get math-based markdown to render properly? When I preview the README, these lines don't seem to be rendered properly, and I haven't found any support in the GitHub documentation.